content/about.en.md

```diff
@@ -12,30 +12,35 @@ As contributors to a common, we collectively do our best to make our community a
 
 In 2016, the Open Government Partnership (OGP) Paris Summit demonstrated the international value of OpenFisca when a small team of OpenFisca France maintainers and OpenFisca Tunisia volunteers managed to model Sénégal's income revenue tax in under 36 hours, with a simulator shipped complete with a web UI during the OGP hackathon, winning the team the first prize.
 
 This led, in 2017, to a joint effort from Etalab and beta.gouv.fr to improve stability, reusability, and ease of contribution. This resulted in major technical improvements, a full rewrite of the documentation, the launch of `openfisca.org` to replace `openfisca.fr`, and the addition of new contributors from other French agencies as well as new international reusers with an experiment from Italy and the city of Barcelona delivering a benefit assessment tool.
 
 In 2018, the product manager that had led the internationalisation efforts [moved](https://www.digital.govt.nz/blog/labplus-lessons-from-matti-schneider-about-service-innovation-at-french-state-incubator-beta-gouv-fr/) to Aotearoa New Zealand, where the recently established [Service Innovation Lab](https://serviceinnovationlab.github.io/projects/legislation-as-code/) was finishing writing its [Better Rules](https://www.digital.govt.nz/blog/what-is-better-rules/) report. This foundational theoretical piece matched perfectly with OpenFisca's implementation of “turning legislation into code”, leading to the Rates Rebate app, the first service in production outside of Europe. Other areas in the Pacific region (Australia, Polynesia, Canada) were progressively exposed to OpenFisca and adopted it.
 
-In 2019, the French Parliament started providing [LexImpact](https://leximpact.an.fr), an impact assessment tool based on OpenFisca, to members of parliament, enabling them to evaluate reforms on the whole French population within a few seconds.
+In 2019, OpenFisca gets the [most innovative open source softaware prize](https://joinup.ec.europa.eu/collection/sharing-and-reuse-it-solutions/sharing-reuse-awards-2019-results#oss-inno) from the Joinup program of the European Commission.  
+This same year, the first [LexImpact](https://leximpact.an.fr) simulator is created. This impact assessment tool based on OpenFisca with the French National Assembly, enables members of parliament to evaluate reforms on test cases or the whole French population within a few seconds.
```

Comment:
```suggestion
On the same year, the first [LexImpact](https://leximpact.an.fr) simulator is created by the French Parliament, enabling elected representatives to evaluate reforms on both test cases and the whole French population within a few seconds.
```

Reply:
Thank you! Taking the update but keeping: 
* `French National Assembly` as its the [official name of the Assemblée nationale in English](https://www2.assemblee-nationale.fr/langues/welcome-to-the-english-website-of-the-french-national-assembly) while the `parliament` includes the Senate.
* `members of parliament` as LexImpact tools are available for members of the Senate that are elected by an indirect election while isn't `elected representatives` more for elections by citizens? 🤔

Reply:
I just wanted to avoid the repetition Parliament / Members of Parliament 🙂 if you keep National Assembly, no problem for “members of parliament”.